lib/capybara/node/actions.rb

```diff
@@ -13,12 +13,13 @@ module Actions
 
       ##
       #
-      # Finds a button or link and clicks it.  See {Capybara::Node::Actions#click_button} and
-      # {Capybara::Node::Actions#click_link} for what locator will match against for each type of element
+      # Finds a button or link and clicks it. See {#click_button} and
+      # {#click_link} for what locator will match against for each type of element.
       #
       # @overload click_link_or_button([locator], **options)
       #   @macro waiting_behavior
-      #   @param [String] locator      See {Capybara::Node::Actions#click_button} and {Capybara::Node::Actions#click_link}
+      #   @param [String] locator      See {#click_button} and {#click_link}
+      #   @param [Hash] options        See {#click_button} and {#click_link}
```

Comment:
This isn't valid -- those options are not available here

Reply:
Oh, sorry. Which do you prefer no `options` or no description or something else?

## No `options`

![image](https://user-images.githubusercontent.com/473530/58218735-044dd000-7d43-11e9-905e-2ccd95c5cfcd.png)

## No description

![image](https://user-images.githubusercontent.com/473530/58218744-13348280-7d43-11e9-9336-0886928268de.png)

Reply:
I prefer the first "No options" one - thanks

Reply:
I trimmed `options`. 👍